HIP-xxxx.md

```diff
@@ -0,0 +1,163 @@
+# HIP-xxxx : Standard for TXT records
+
+```
+Number:  HIP-xxxx
+Title:   Standard for TXT records
+Type:    Standards
+Status:  Draft
+Authors: 0xStefan <https://twitter.com/0xStefan>
+Created: 2022-02-22
+```
+
+## Abstract
+
+A proposal for TXT record standards to provide on-chain data for a domain.
+
+## Motivation
+
+#### Domain Parking
+
+There's currently no way to know if a domain is for sale by the owner or not.
+A domain seller has no reliable way to provide that information.
+
+#### Identity / Web3 Profile
+
+Websites using Sign in with Handshake can use and render the TXT record data, e.g. as profile information. The domain owner is still in control of the data.
+
+## TXT Standards Definition
+
+For the above to work we need standards so any website can properly render the TXT records. The records should be flexible, short and easy to understand. Since it's impossible to validate the data via HSD. Therefore TXT records should be provided mostly without any prefix like "mailto:", "tel:" or "https://www.twitter.com/".
+
+The goal is to let the app or website decide how to render the information. It also decreases the data saved on-chain.
+
+#### Domain Parking
+
+The parking TXT entry indicates that the domain is for sale.
+
+```
+TXT parking=[String]
+```
+
+#### Contact or Profile Information
+
+```
+TXT email=[Email]
+TXT phone=[Tel]
+```
+
+```
+TXT discord=[String]
+TXT facebook=[String]
+TXT github=[String]
+TXT instagram=[String]
+TXT linkedin=[String]
+TXT telegram=[String]
+TXT tiktok=[String]
+TXT twitter=[String]
+TXT weixin=[String]
+TXT whatsapp=[Tel]
+TXT youtube=[String]
+```
+
+#### Avatar
+
+```
+TXT avatar=[ URL | skylink | ipfsLink ]
+```
+
+#### Domain Caption
+
+A caption is especially useful for identities or as a sales pitch. It can also be used to add more context to a domain (e.g. NFT's).
+
+```
+TXT caption=[String]
+```
+
+#### Payment / Wallet Alias
+
+Although [HIP-0002](https://github.com/handshake-org/HIPs/blob/master/HIP-0002.md) is a much better approach, TXT entries could also be used to provide static wallet information.
+
+```
+TXT wallet=[Chain]:[Address]
+```
+
+## Valid TXT Standard Examples
+
+#### Parking Examples
+
+```
+TXT parking
+TXT parking=10000 HNS
+TXT parking=USD 1000
+TXT parking=USD 999.99
+TXT parking=999,99 EUR
+TXT parking=$199 (Old price $329)
+TXT parking=Please send an offer via email
```

Comment:
I don't think "parking" is the correct key. 

Parking.Sinpapeles has 2 main features: 
1. Landing page (aka parking page): visitors can access the TLD and find more info about it.
2. Index and list all domains available.

There are multiple website/projects doing the second point (including Niami and hns.nameboard), but AFAIK, only parking.sinpapeles is doing the first point. And that is the "parking".

Reply:
Maybe can be called `forsale`, `sale`, `listing`, or something similar.

I like the idea of being able to indicate that a name is available to buy without mentioning anything. Just a note that not having an `=` might be unintuitive while parsing by apps since this looks like the only field that can do it?

Honestly, I would prefer having 2 fields, both optional:
- a dedicated field for value (tuple of number and unit/currency) - if blank, then not a public amount
- a dedicated field for marketplace URI with mandatory protocol: could be http, https, mailto, etc. - if blank, there's still other fields that are displayed under **Contact or Profile Information**

Reply:
TBH I just used `parking=` because I thought it needed to be included due to past discussions. ;)
I also think it's unintuitive.

Must say I really like `listing`. It makes sense. 
I suggest we do it like that:

`listing price=`
`listing url=`

By using `listing XXX=` we also have the option to add more attributes in the future.